templates/app/views/layouts/storefront.html.erb.tt

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="<%%= I18n.locale %>">
+  <head>
+    <meta charset="utf-8">
+    <title><%%= title %></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%%== meta_data_tags %>
+    <%%= canonical_tag(current_store.url) %>
+    <%%= csrf_meta_tags %>
+
+    <%%= javascript_include_tag 'solidus_starter_frontend' %>
+    <%%= stylesheet_link_tag 'solidus_starter_frontend', media: 'screen' %>
+
+<% if Rails.application.respond_to? :importmap -%>
+    <%%= javascript_importmap_tags %>
+<% end -%>
+
+    <%%= favicon_link_tag 'favicon.ico' %>
+    <%%= yield :head %>
+  </head>
+  <body x class="<%%= body_class %>" id="<%= @body_id || 'default' %>">
```

Comment:
@kennyadsl I notice that the x attribute was added in this commit. Is it a typo?

Reply:
Ops, yes, it was a test I was doing to understand if my changes were immediately reflected when rebuilding the sandbox. BTW, they weren't: I needed to create a commit to see those changes applied.

Thanks a lot for spotting this.